src/einsteinpy/coordinates/velocity.py

```diff
@@ -61,6 +61,18 @@ def si_values(self):
             )
         )
 
+    def vel_coord(self):
```

Comment:
`def velocities(return_np = False):` can be a better name. When `False`, you can return a list `[self.v_x, self.v_y, self.v_z]`.
When `True`, you can return `self.si_values()[3:]`. How does it sound?

There is no point returning strings, maybe the user would want to do something with the numbers.

Also, we should try to avoid `_`s as much as possible while naming functions :D

Reply:
yeah that would be better!

Reply:
for returning string, a separate function can be made. It's not quite a concern!!

Reply:
I created a new PR for velocity. Shall I revert the last commit?

Reply:
no need. we'll clean this up!